third_party/terraform/utils/config.go.erb

```diff
@@ -705,6 +707,20 @@ func (c *Config) NewHealthcareClient(userAgent string) *healthcare.Service {
 	return clientHealthcare
 }
 
+func (c *Config) NewHealthcareBetaClient(userAgent string) *healthcareBeta.Service {
```

Comment:
If we generate support for the IAM resources we shouldn't need this new client

Reply:
I added this because unlike other healthcare apis in ga, this one is in beta.

Reply:
Yep, but this is only used for the IAM resource, which can be generated without using the client

Reply:
And I also use it in third_party/terraform/tests/resource_healthcare_consent_store_test.go.erb